cogs/vxtwitconverter/helpers.py

```diff
@@ -24,23 +52,25 @@ def convert_to_vx_twitter_url(embeds: list[Embed]):
     return vxtwitter_urls
 
 
-def urls_to_string(vx_twit_links: list[str]):
+def urls_to_string(links: list[str], socialMedia: SocialMedia):
     """
     Parameters
     ----------
-    vx_twit_links: list of urls
+    links: List[str]
+        A list of urls
+    socialMedia: SocialMedia
+        The social media to replace.
 
     Returns
     -------
         Formatted output
     """
-
     return "".join(
         [
             "OwO what's this?\n",
-            "*notices your terrible twitter embeds*\n",
+            f"*notices your terrible {socialMedia.value} embeds*\n",
             "Here's a better alternative:\n",
-            "\n".join(vx_twit_links),
+            "\n".join(links),
         ]
```

Comment:
Might as well optimize this:
```suggestion
    return "\n".join(
        (
            "OwO what's this?",
            f"*notices your terrible {socialMedia.value} embeds*",
            "Here's a better alternative:",
            *links,
        )
```